src/style.css

```
@@ -1,3 +1,69 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
```

```suggestion
.header {
  padding: 0 50px

  display: flex;
```
and just add the padding here

---

the padding on the right did not appear